vaccine_feed_ingest/runners/us/physicians_immediate/normalize.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import pathlib
+import pprint
+import sys
+from typing import Dict, List, Optional, OrderedDict
+
+import usaddress
+from vaccine_feed_ingest_schema import location as schema
+
+from vaccine_feed_ingest.utils.log import getLogger
+
+logger = getLogger(__file__)
+
+Site = Dict[str, str]
+
+
+def get_site_id(site: Site) -> str:
+    return f"us_physicians_immediate:{site['table_id']}_{site['row_id']}"
+
+
+def infer_address(site: Site) -> schema.Address:
+    """The source data does not separate address components into separate fields.
+    This function imports the probabilistic parser from `usaddress` to make sense
+    of the scraped address data. Some extra patches are applied to fix errors in this
+    specfic dataset.
+    """
+
+    parsed_address, address_type = usaddress.tag(site["address"])
+    assert (
+        address_type == "Street Address"
+    ), f"Invariant: address normalization only works for addresses of type 'Street Address', not {address_type}"
+    patched_address = apply_address_fixups(parsed_address)
+
+    # Building up kwargs lets us easily omit fields for which we have no data.
+    # In particular, many sites don't provide a zip code
+    address_kwargs = {
+        # "street1",
+        # "city",
+        # "state",
+        # "zip"
+    }
+    street_buffer: List[str] = []
+    while len(patched_address) > 0:
+        component, value = patched_address.popitem(last=False)
+        if component == "PlaceName":
+            address_kwargs["city"] = value
+        elif component == "StateName":
+            address_kwargs["state"] = value
+        elif component == "ZipCode":
+            address_kwargs["zip"] = value
+        else:
+            street_buffer.append(value)
+    address_kwargs["street1"] = " ".join(street_buffer)
+
+    result = schema.Address(**address_kwargs)
+    logger.info(f"Parsed address: {site['address']}")
+    logger.info(f"Normalized address:{pprint.pformat(result)}")
```

Comment:
Please disable this logging (maybe just comment them out or downgrade them to debug).